languages/r36/template/src/Algorithm.r

```diff
@@ -0,0 +1,36 @@
+library(algorithmia)
+# API calls will begin at the algorithm() method, with the request body passed as 'input'
+# For more details, see algorithmia.com/developers/algorithm-development/languages
+
+algorithm <- function(input) {
+  paste("hello", input)
+}
+
+# Here is an example of an advanced form of an algorithm function, using a load function (like the example below)
+# -- ADVANCED ALGORITHM USAGE --
+#   algorithm <- function(input, keras_model) {
+#   ....
+#   prediction <- keras_model.predict(input)
+#   result <- list(class <- prediction[0], confidence <- prediction[1])
+#   result
+# }
+
+
+load <- function() {
+  # Here you can optionally define a function called when the algorithm is loaded.
```

Comment:
do you want to be explicit and return null?

Reply:
sure I can add that